doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -6,6 +6,7 @@
 
 **Features:**
 
+- Add bazel asan, usan, tsan build config [#1547](https://github.com/eclipse-iceoryx/iceoryx/issues/1547)
```

Comment:
For future contributions, please always add new entries at the end of the changelog :)

Reply:
I got it.